src/core/Akka.Tests/Routing/ScatterGatherFirstCompletedSpec.cs

```diff
@@ -112,27 +159,41 @@ protected override void OnReceive(object message)
                 }
                 else
                 {
-                    Thread.Sleep(100*_id);
+                    Thread.Sleep(100 * _id);
                     Sender.Tell(_id);
                 }
             }
         }
 
-        [Fact]
-        public void Scatter_gather_router_must_return_response_even_if_one_of_the_actors_has_stopped()
+        public class DelayStopActor : UntypedActor
         {
-            var shutdownLatch = new TestLatch(1);
-            var actor1 = Sys.ActorOf(Props.Create(() => new StopActor(1)));
-            var actor2 = Sys.ActorOf(Props.Create(() => new StopActor(14)));
-            var paths = new []{actor1,actor2};
-            var routedActor = Sys.ActorOf(new ScatterGatherFirstCompletedGroup(paths, TimeSpan.FromSeconds(3)).Props());
+            private int _id;
+            private readonly TimeSpan _waitTime;
 
-            routedActor.Tell(new Broadcast(new Stop(1)));
-            shutdownLatch.Open();
-            var res = routedActor.Ask<int>(new Broadcast(0), TimeSpan.FromSeconds(10));
-            res.Wait();
-            res.Result.ShouldBe(14);
+            public DelayStopActor(int id, TimeSpan waitTime)
```

Comment:
Prob don't need this. I saw that the other actor delays based on id so i can just use that functionality